doc/Telco-PaaS-definition/TelcoPaaSDefChart.md

```diff
@@ -20,7 +20,7 @@
 |17|Firewall||||PaaS-Application support||
 |18|DPI||||PaaS-Application support||
 |19|NAT||||PaaS-Application support||
-|20|Hardware acceleration||||PaaS-Application support||
+|20|Hardware acceleration|(1) Hardware feature detect: NFD (2) Nic: SRIOV, FPGA (3) QAT: security and data compression (3) AI workload: FPGA, VPU (4) CPU core management: CMK|(1) NFD: https://github.com/kubernetes-sigs/node-feature-discovery (2) CMK: https://github.com/intel/CPU-Manager-for-Kubernetes (3 )QAT/FPGA device plugin: https://github.com/intel/intel-device-plugins-for-kubernetes (4)SRIOV device plugin: https://github.com/intel/sriov-network-device-plugin||PaaS-Application support||
```

Comment:
CMK does not provide an overall CPU management, rather it provides support to allocate the cores within a container. For full fledged CPU management [CPU Pooler](https://github.com/nokia/CPU-Pooler/) is needed.